.github/workflows/markdown-links-config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ignorePatterns": [
+    {
+        "pattern": "^http://localhost"
+    },
+    {
+        "pattern": "^https://twitter.com/"
+    },
+    {
+        "pattern": "^gaurav-nelson/"
+    }
+  ],
+  "aliveStatusCodes": [403, 429, 200],
+  "httpHeaders": [
+    {
+      "urls": [
+        "https://docs.github.com/",
+        "https://opensource.org/"
+      ],
+      "headers": {
+        "Accept-Encoding": "zstd, br, gzip, deflate"
```

Comment:
Why does this matter?

Reply:
defensive configuration for any potential future bugs because of these
https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/136
https://github.com/tcort/markdown-link-check/issues/201#issuecomment-1110242146
https://github.com/github/docs/issues/17358#issuecomment-1350014801